frontend/src/properties/propertysections/ChartWidgetPropertySection.tsx

```diff
@@ -34,7 +28,7 @@ export const ChartWidgetPropertySection = ({ widget, subscribers }: ChartWidgetP
   } else if (isPieChart(chart)) {
```

Comment:
`chartComponent` three lines above should be typed with `JSX.Element | null`

Reply:
ok

Reply:
ok